src/Avalonia.Controls/NumericUpDown/NumericUpDown.cs

```diff
@@ -612,6 +649,10 @@ protected virtual void RaiseValueChangedEvent(decimal? oldValue, decimal? newVal
         /// <returns></returns>
         private string? ConvertValueToText()
         {
+            if (TextConverter != null)
+            {
+                return TextConverter.ConvertBack(Value, typeof(string), null, _cultureInfo ?? CultureInfo.CurrentUICulture)?.ToString();
```

Comment:
`_cultureInfo` isn't set anywhere, so it's always `null`. I believe it's a leftover from the removed `CultureInfo` property that was replaced by `NumberFormat`, so it shouldn't be used and should be removed.


Also, it seems that all of our  `Converter`.`Convert`/`ConvertBack` calls [use CultureInfo.CurrentCulture](https://github.com/AvaloniaUI/Avalonia/search?q=currentculture) instead of `CurrentUICulture`. Not sure why that's the case. @grokys  ?

Reply:
Ah, I guess that's consistent with WPF where CurrentUICulture is only used for satellite assemblies with resources.

Reply:
Changed to CurrentCulture.